app/components/modal/order/order-cart.html

```diff
@@ -0,0 +1,65 @@
+<div class="modal-header">
+    <button type="button" class="close" ng-click="orderModalCtrl.modalCancel()">&times;</button>
+    <h3 ng-hide="orderModalCtrl.order == '' || orderModalCtrl.order == undefined" class="modal-title" id="modal-title">
+        Confirm Order</h3>
+</div>
+<div class="modal-body" id="modal-body">
+
+    <table class="table table-condensed table-hover table-striped table-bordered ">
+        <caption ng-hide="orderModalCtrl.order == '' || orderModalCtrl.order == undefined">
+            <h4><u>Order List</u></h4>
+        </caption>
+        <thead ng-hide="orderModalCtrl.order == '' || orderModalCtrl.order == undefined">
+        <tr>
+            <th>S/N</th>
+            <th>Restaurant</th>
+            <th>Food</th>
+            <th>Unit Price</th>
+            <th>Quantity</th>
+            <th>Sub Total</th>
+            <th>Remove</th>
+        </tr>
+        </thead>
+        <tbody>
+        <h3 ng-show="orderModalCtrl.order == '' || orderModalCtrl.order == undefined" colspan="7"
+            style="text-align: center; color: red;"><strong>You have no orders.</strong>
+        </h3>
+        <tr ng-repeat="food in orderModalCtrl.order">
+            <td>{{$index + 1}}</td>
+            <td>{{food.restaurantName}}</td>
+            <td>{{food.name}}</td>
+            <td>{{food.price}}</td>
+            <td>
+                <button class="btn btn-sm num-control" ng-click="orderModalCtrl.quantityDecrease($index,food)"
+                        ng-disabled="food.quantity <= 1"><span
+                        class="glyphicon glyphicon-minus"></span>
+                </button>
+                &nbsp;&nbsp;
+                {{food.quantity}}
+                &nbsp;&nbsp;
+                <button class="btn btn-sm num-control" ng-click="orderModalCtrl.quantityIncrease($index,food)"><span
+                        class="glyphicon glyphicon-plus"></span></button>
+            </td>
+            <td style="text-align: center;">{{food.price * food.quantity}}</td>
+            <td style="text-align: center;">
+                <button class="btn btn-sm" ng-click="orderModalCtrl.deleteOrder(food)"><span
+                        class="glyphicon glyphicon-remove"></span></button>
+            </td>
+        </tr>
+        <tr ng-hide="orderModalCtrl.getTotal()<=0">
```

Comment:
"orderModalCtrl.getTotal()<=0" does getTotal() goes to negative value ?, why you are using less than equal to zero ?, you can instead use ng-show="orderModalCtrl.getTotal()" so, if the getTotal() return 0 then it will not be render, and if getTotal() return 1 or more than it will be render.